install-jx.sh

```diff
@@ -4,18 +4,21 @@ JX_VERSION=1.3.971
 
 function install_dependencies() {
     mkdir -p ~/bin
+    echo "Downloading & installing Jenkins X ${JX_VERSION}"
     curl -sL https://github.com/jenkins-x/jx/releases/download/v${JX_VERSION}/jx-linux-amd64.tar.gz \
     | tar xvz -C ~/bin
+    echo "Installed"
 }
 
 function add_path_to_bashrc() {
     if grep -q PATH ~/.bashrc; then
-        echo ""
+        echo "Already on path"
     else
         echo "export PATH=$PATH:$HOME/bin:$HOME/.jx/bin" >> ~/.bashrc
-        echo "export PATH=$PATH:$HOME/bin:$HOME/.jx/bin"
+        export PATH=$PATH:$HOME/bin:$HOME/.jx/bin
+        jx --help
```

Comment:
could remove/move this. was just replicating previous functionality

Reply:
Could source the ~/.bashrc file here maybe - would remove duplicate PATH export